src/includes/modal/checkpaypsp.pug

```diff
@@ -1,6 +0,0 @@
-.modalwindow(id="modal-checkpaypsp")
```

Comment:
is it replaced by `checkwhypsp.pug`? right? so it needs to be deleted from `src/check.pug`